code_challenges/hacker_rank/exceptions.py

```diff
@@ -0,0 +1,24 @@
+def find_the_exception(the_values):
+    result = []
+    for first, second in the_values:
+        try:
+            result.append(str(int(int(first) / int(second))))
```

Comment:
Casting to string here is what I had in mind. 


In `str(int(int(first) / int(second)))` if the intention is to drop decimal places with the surrounding `int`.  Would integer division work ( `//`) instead?